centaur/src/main/resources/standardTestCases/blob_md5/blob_md5.inputs.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
These Centaur files do not have an associated `.test` file yet so they are not actually running, but I did think it was time to promote them from Slack/local disk to source control.

Reply:
There are a bunch of places we do this with GCP, specifically the buckets in `broad-dsde-cromwell-dev`.
https://github.com/broadinstitute/cromwell/blob/d16138a2175d68f8d83bac29257c797202b5bf2c/centaur/src/main/resources/standardTestCases/input_localization/localize_file_larger_than_disk_space.wdl#L20
We don't have to go down the same route for Azure, but I'd prefer to defer that discussion to when we set up integration testing. For now, I genericized it since I think we truly do have no idea what account we'll be in then.